src/test/proto/compliance.proto

```diff
@@ -80,6 +80,44 @@ service Compliance {
       get: "/v1beta1/repeat/{info.f_string=first/*}/{info.f_child.f_string=second/**}:pathtrailingresource"
     };
   }
+
+  // This method requests an enum value from the server. Depending on the contents of EnumRequest, the enum value returned will be a known enum declared in the
```

Comment:
general question - is there a reason this proto doesn't match https://github.com/googleapis/gapic-showcase/blob/main/schema/google/showcase/v1beta1/compliance.proto exactly?

Reply:
This was way before I joined, but I guess what happened was that the showcase protos have a lot of edge cases that we want to use, so instead of creating protos from scratch for our test cases, we just copied over these showcase protos and use them as they are. However, protos in showcase evolve over time and we didn't update them regularly.